tests/test_readme.py

```
@@ -0,0 +1,180 @@
+"""Test suite for README.md validation."""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)